src/dvu/gfx/visualization.ts

```diff
@@ -19,12 +19,12 @@ export class CompositeVisualization extends PictureCommand {
     super(this.name, this.getPictureCommandInterface())
   }
 
-  execute(context: PictureContext, scope: Scope = new Scope()): Picture {
+  execute(scope: Scope = new Scope()): Picture {
```

Comment:
Eventually we'll receive 'arguments' as `data` here, it'll not be context but the first argument is important for us since it'll drive the dynamic-ism part later.